themes/vocabulary_theme/templates/blocks/recent-posts.html

```diff
@@ -0,0 +1,41 @@
+{% from "macros/author_name.html" import render_author_name %}
+
+<section class="recent-posts">
+  <div class="container">
+    <div class="level">
+      <h4 class="is-paddingless level-left">
+        {{ this.title }}
+      </h4>
+      <span class="level-right">
+        <a class="post-links" href="/blog">See all posts <i class="icon angle-right"></i></a>
+      </span>
+    </div>
+    <div class="columns">
+      {% for post in site.query('/blog/entries') %}
+        {% if loop.index <= 3 %}
+          {% set author = post.parent.parent.children.get('authors').children.get(post.author) %}
+          <article class="card entry-post horizontal no-border blog-entry column is-one-third">
```

Comment:
I would try to keep separated the component from the column container. For instance:
```
<div class="columns">
  <div class="column is-one-third">
    <article class="card entry-post horizontal no-border blog-entry">
    </article>
  </div>
```

I think it makes more sense in terms of content hierarchy

Reply:
Makes sense!